src/drivers/imx/ipc.c

```diff
@@ -107,7 +107,7 @@ static void irq_handler(void *arg)
 
 void ipc_platform_do_cmd(struct ipc *ipc)
 {
-	struct ipc_data *iipc = ipc_get_drvdata(ipc);
+	/* Use struct ipc_data *iipc = ipc_get_drvdata(ipc); if needed */
```

Comment:
I think it's better to just remove that instead of keeping in comment.

Reply:
I kept it intentionally as a hint for re-implementation when needed again. I am going to write a short piece of documentation about the IPC flows and private data use by platform specific IPC layer. Then it will become redundant indeed.

Reply:
I disagree. IMHO we have version control for that, so we shouldn't comment code just to use it later. How-tos should be in docs. But I can live with that, if you are going to remove it anyway.

Reply:
OK, lines previously commented out removed removed from the code. I added a short piece of documentation to `platform_ipc_init()` instead. 

Btw there are `ipc_platform_...()` and `platform_ipc_...()` names in the code. I am not sure what is the reason for different prefix/scope name order. Need to check if the former one improves code locality when used together with common `ipc_..()` code (depends on the linker's sorting method). Perhaps sth to unify later.